editor/resources/meta.edn

```diff
@@ -252,10 +252,12 @@
    :path ["physics" "max_fixed_timesteps"]},
   {:type :string,
    :help
-   "which filtering to use for min filtering, linear (default) or nearest",
+   "which filtering to use for min filtering, linear by default",
    :default "linear",
    :path ["graphics" "default_texture_min_filter"]
-   :options [["linear" "linear"] ["nearest" "nearest"]]}
+   :options [["linear" "linear"] ["nearest" "nearest"] ["nearest_mipmap_nearest" "nearest mipmap nearest"]
+   									["nearest_mipmap_linear" "nearest mipmap linear"] ["linear_mipmap_nearest" "linear mipmap nearest"]
+   									["linear_mipmap_linear" "linear mipmap linear"]]}
```

Comment:
Please use spaces for indentation in editor files. Also, I think we could split these across several lines so that we have one setting-pair per line now that there are many.

Reply:
thx, done.